README.md

```diff
@@ -41,39 +41,41 @@ This tool expects to run in an environment with the following attributes:
 
 Requesting a code review:
 
-    git appraise request
+    git review request
 
 Pushing code reviews to a remote:
 
-    git appraise push [<remote>]
+    git review push [<remote>]
 
 Pulling code reviews from a remote:
 
-    git appraise pull [<remote>]
+    git review pull [<remote>]
 
 Listing open code reviews:
 
-    git appraise list
+    git review
+    git review --list
 
 Showing the status of the current review, including comments:
 
-    git appraise show
+    git review show
 
 Showing the diff of a review:
 
-    git appraise show --diff [--diff-opts "<diff-options>"] [<review-hash>]
+    git review show -p/-u/--patch [--diff-opts "<diff-options>"] [<review-hash>]
 
 Commenting on a review:
 
-    git appraise comment -m "<message>" [-f <file> [-l <line>]] [<review-hash>]
+    git review comment -m "<message>" [-f <file> [-l <line>]] [<review-hash>]
 
 Accepting the changes in a review:
 
-    git appraise accept [-m "<message>"] [<review-hash>]
+    git review accept [-m "<message>"] [<review-hash>]
 
 Submitting the current review:
 
-    git appraise submit [--merge | --rebase]
+    git review merge [--ff | --no-ff | --ff-only ]
+    git review rebase
```

Comment:
I think the reason why `submit` doesn't appear in the git world (in this context at least) is because git itself doesn't deal with code reviews where as git-appraise does. Although the code is very much tied to the review itself, it is the review we are submitting and as a result of that the code is then merged.

If you think about it just based on the terms alone, "merging a review" doesn't really make much sense, where as submitting one does. Again, it's contextual.